.pipeline/scripts/get-maven-excludes.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import argparse
+
+def write_maven_excludes_github_output(filter_key, filter_value):
+    with open("module-inventory.json", "r") as file:
+        module_inventory = json.load(file)
+
+        filtered_inventory = filter(lambda x: x[filter_key] == filter_value, module_inventory)
```

Comment:
This comparison might become a problem when filtering for non-string properties (e.g. boolean values).

```suggestion
        filtered_inventory = filter(lambda x: x[filter_key] == str(filter_value), module_inventory)
```

Reply:
Good point, but I guess you meant it like this right?
```suggestion
        filtered_inventory = filter(lambda x: str(x[filter_key]) == filter_value, module_inventory)
```
I mean `filter_values` should always be a string as it is a parameter and argparse should parse it as string by default, but the value in the modules-inventory might be a non string property like the `excludeFromBlackDuckScan` property.

Reply:
Yes, you're right!